nncf/tensorflow/graph/model_transformer.py

```diff
@@ -302,19 +308,21 @@ def _insert_layers_after(self, layer_name: str, instance_idx: int, output_port_i
 
         for config in layer_configs:
             if functional_model:
-                self._insert_layer_after_functional(layer_name, instance_idx, config)
+                self._insert_layer_after_functional(layer_name, instance_idx, config, inserted_instance_idx)
             else:
                 self._insert_layer_after_sequential(layer_name, config)
 
-    def _insert_layer_after_functional(self, layer_name: str, instance_idx: int, layer_config: Dict):
+    def _insert_layer_after_functional(self, layer_name: str, instance_idx: int, layer_config: Dict,
+                                       inserted_instance_idx: int):
```

Comment:
Propose to rename `inserted_instance_idx` to `insert_with_instance_idx` here and everywhere.

Reply:
Renamed